lib/engine/step/g_1856/track.rb

```diff
@@ -35,6 +35,12 @@ def old_paths_are_preserved(old_paths, new_paths)
 
           old_paths.all? { |path| new_paths.any? { |p| path <= p } }
         end
+
+        def process_lay_tile(action)
+          super
+          # TODO: Replace 'true' with @loading check & implement auto action usage
+          @game.destination_check # if true
```

Comment:
don't forget to do this or better to just comment it out for now

Reply:
I'm going to see if I can get this to work in this PR

Reply:
Adding the auto-action to handle destinating/escrow-release looks like it would be complicated enough to warrant a followup PR.